ntoskrnl/io/iomgr/iomgr.c

```diff
@@ -584,18 +584,10 @@ IoInitSystem(IN PLOADER_PARAMETER_BLOCK LoaderBlock)
         return FALSE;
     }
 
-    // the disk subsystem is initialized here and the SystemRoot is set too
-    // we can finally load other drivers from the boot volume
+    /* The disk subsystem is initialized here and the SystemRoot is set too.
```

Comment:
Please put formatting / annotation / spelling fixes into a separate commit to keep the actual changes as minimal as possible.

Reply:
Will do :+1: